index.html

```diff
@@ -38,5 +38,115 @@ <h4 class="connect">LET'S CONNECT</h4>
+                <ul class="client">
+                    <li class="worktime tag"><b>html</b></li>
+                    <li class="worktime tag"><b>css</b></li>
+                    <li class="worktime tag"><b>javaScript</b></li>
+                </ul>
+                <button class="cardButton" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="card">
+            <img class="cardsection" src="images/forthwork.png" alt="SnapshootPortfolio">
+            <div class="cardsection">
+                <h4>Multi-Post Stories</h4> 
+                <ul class="client">
+                    <li class="worktime"><b>CANOPY</b></li>
+                    <li class="worktime"><img src="images/Counter.png"></li>
+                    <li class="worktime worktitle">Back End Dev</li>
+                    <li class="worktime"><img src="images/Counter.png"></li>
+                    <li class="worktitle">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="client">
+                    <li class="worktime tag"><b>html</b></li>
+                    <li class="worktime tag"><b>css</b></li>
+                    <li class="worktime tag"><b>javaScript</b></li>
+                </ul>
+                <button class="cardButton" type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section class="aboutMe">
+        <h2>About <br>Myself</h2>
+        <p class="resume">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <h4 class="connect">LET'S CONNECT</h4>
+        <div class="social">
+            <ul class="personalMedia">
+                <li><a class="media" href="https://twitter.com/Santiag24209785"><img src="images/twitter.png"></a></li>
+                <li><a class="media" href="https://www.linkedin.com/in/santiago-munoz-0b2b1a260/"><img src="images/linkedin.png"></a></li>
+                <li><a class="media" href="https://github.com/smunoz1988"><img src="images/github.png"></a></li>
+            </ul>
+        </div>
+        <button class="resumeButton" type="button">Get my resume</button>
+        <div>
+            <ul class="categories">
+                <li class="atribute">Languajes</li>
+                <ul class="languaje">
+                    <li class="js"><img src="images/js.png"><p>JavaScript</p></li>
+                    <li class="html"><img src="images/html.png"><p>HTML</p></li>
+                    <li class="css"><img src="images/css.png"><p>CSS</p></li>
+                </ul>
+                <li class="atribute">Frameworks</li>
+                <hr>
+                <li class="atribute">Skills</li>
+                <hr>
+            </ul>
```

Comment:
- [ ] You have done a great job 👍so far in making your project look like the Figma design. Please🙏 add the missing elements that are found in the Figma design, which will make your project look like the original Figma design. This is a good indicator that the section is clickable.
![image](https://user-images.githubusercontent.com/63445083/219758026-df3707b5-1f08-4cf6-b717-eb22d21bd474.png)